src/native/external/README.md

```diff
@@ -0,0 +1,79 @@
+# Native external libraries
+
+This folder contains the source code of vendored third party native libraries that are used as dependencies for dotnet/runtime.
+
+We call "vendored libraries" those from which we keep a copy of their source code in this repository, and those copies are built alongside the rest of our repository. The main document describing vendored libraries can be found here: https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Strategy-For-External-Source.md
```

Comment:
```suggestion
Vendored libraries are copies of third-party dependencies that are included in the project repository. These copies are built alongside the rest of the repository. The document describing our approach for vendored libraries can be found here: https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Strategy-For-External-Source.md
```
We have not invented the terms "vendoring" or "vendored libraries". It is a standard industry term, so we should not say "We call ...". Everybody calls it like that.